packages/suite-web/src/static/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Side note: I think correct naming of this file is now site.webmanifest

Reply:
`manifest.json` is still valid but there's a dedicated extension for it now. I will update it to `.webmanifest`.

Reply:
f2775a8